app/views/admin/r_and_r_items/index.html.erb

```diff
@@ -81,6 +81,7 @@
         <th scope="col" class="d-none d-md-table-cell" >Bib number</th>
         <th scope="col" class="d-none d-md-table-cell" >Accession number</th>
         <th scope="col" class="d-none d-md-table-cell" >Object ID</th>
+        <th scope="col" class="d-none d-md-table-cell" >Location </th>
```

Comment:
There's room in the table for "Location", you tested this?

On Dig Queue Item, I put location under "title" instead, because I thought location can get quite long. 

But I guess R&R and Dig Queue list view just continue to diverge, I guess that's just what happens when they aren't DRY. 

In general, have you tested the CSS/html for this table with what happens when values get lengthy?  Make sure it handles them okay without totally falling apart?

Reply:
Good call. I limited the width for the title and notes field, which can get long. We can always adjust further once we have some real values in the table, of course. But the metadata columns fit just fine. They get hidden on narrow screen widths of course, but this part of the admin won't be used on narrow screen widths, in practice.